platform/pulp/platform/models/consumer.py

```diff
@@ -0,0 +1,67 @@
+"""
+Django models related to content consumers.
+"""
+
+from django.contrib.contenttypes.fields import GenericRelation
+from django.db.models import TextField, ForeignKey, ManyToManyField, CASCADE
+
+from pulp.platform.models import Model, Notes
+from pulp.platform.models.repository import Distributor
```

Comment:
Agreed.  I vote for importing from the package:
```
from pulp.platform.models import Model, Notes, Distributor
```

... just missed it when adding the new models.